core-web/libs/block-editor/src/lib/components/dot-block-editor/dot-block-editor.component.scss

```diff
@@ -299,6 +299,45 @@
                 }
             }
         }
+
+        .ai-content-container {
+            &.ProseMirror-selectednode {
+                background-color: $color-palette-primary-op-20;
+                border: 1px solid $color-palette-primary-300;
+            }
+        }
+
+        .loader-style {
+            display: flex;
+            justify-content: center;
+            flex-direction: column;
+            align-items: center;
+            min-height: 12.5rem;
+            min-width: 25rem;
+            width: 25.5625rem;
+            height: 17.375rem;
+            padding: 0.5rem;
+            border-radius: 0.5rem;
```

Comment:
We have variables for border-radius.

Reply:
https://github.com/nikolatrajkovic24/core/blob/pr/nikolatrajkovic24/26405/core-web/libs/dotcms-scss/shared/_common.scss#L1-L5